mbkp.sh

```diff
@@ -120,7 +120,7 @@ $SSH_STR "console clear-history"
 function fn_mikrotik_fixtime {
 # Function for setting ntp client
 
-$SSH_STR "ip cloud set update-time=no; system ntp client set primary-ntp=0.0.0.0 secondary-ntp=0.0.0.0 enabled=yes server-dns-names=pool.ntp.org"
+$SSH_STR "ip cloud set update-time=no; system ntp client set primary-ntp=200.160.0.8 secondary-ntp=200.189.40.8 enabled=yes server-dns-names=pool.ntp.org"
```

Comment:
So, directive primary-ntp and secondary-ntp will configure SNTP client to communicate with two servers only. Using directive server-dns-names (Comma separated domain name list default: ) we can set ntp pool which is most reliable to receive NTP.
One thing that we need to keep in mind - it could not work without DNS configured.

It works well:
[rmgmt01@vl-nmr01] > system ntp client print
enabled: yes
primary-ntp: 0.0.0.0
secondary-ntp: 0.0.0.0
server-dns-names: 0.ru.pool.ntp.org,1.ru.pool.ntp.org,2.ru.pool.ntp.org,3.ru.pool.ntp.org
mode: unicast
poll-interval: 15m
active-server: 94.247.111.10
last-update-from: 94.247.111.10
last-update-before: 16m33s430ms
last-adjustment: 5ms477us
last-bad-packet-from: 10.1.201.249
last-bad-packet-before: 12m8s350ms
last-bad-packet-reason: server-ip-mismatch